src/Tests/Microsoft.Diagnostics.Monitoring.TestCommon/CommonTestTimeouts.cs

```diff
@@ -42,5 +42,35 @@ public static class CommonTestTimeouts
         /// Dumps (especially full dumps) can be quite large and take a significant amount of time to transfer.
         /// </remarks>
         public static readonly TimeSpan DumpTimeout = TimeSpan.FromMinutes(3);
+
+        /// <summary>
+        /// Default timeout for logs collection.
+        /// </summary>
+        public static readonly TimeSpan LogsTimeout = TimeSpan.FromMinutes(3);
```

Comment:
LogsTimeout and LogsDuration make sense here. The others should move back to where they were previously since they are not shared.